### PR TITLE
feat: add ProcessingStep (from Spectra package)

### PR DESCRIPTION
**Pull request by jorainer**

### CODE REVIEW

File: man/protgenerics.Rd [hunk 1]
```diff
@@ -102,6 +102,7 @@
 \alias{compounds}
 \alias{filterIntensity}
 \alias{alignRt}
+\alias{quantify}
```
Comment (lgatto) [inline man/protgenerics.Rd:105]: thanks for adding this one!

**Comment by lgatto** [comment]
Do you want me to push to Bioc already?

**Comment by jorainer** [comment]
yes please, than I can proceed pushing also `Spectra` - for `MSnbase` I made already a PR

**Comment by lgatto** [comment]
Done

**Pull request merged**